src/common/components/organisms/EditorPanel.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { Dispatch, SetStateAction, useState } from "react";
+import React, { Dispatch, SetStateAction } from "react";
```

Comment:
Is this needed? It looks like we don't use any of these imports except `React` itself for the `.tsx` compliation

Reply:
hey @hiporox ! we use `Dispatch` and `SetStateAction` on lines 12, 13 in this file